subsys/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ endif()
 
 add_subdirectory_ifdef(CONFIG_IS_SECURE_BOOTLOADER bootloader)
 add_subdirectory_ifdef(CONFIG_SECURE_BOOT bootloader/image)
+add_subdirectory_ifdef(CONFIG_SECURE_BOOT_CRYPTO_CLIENT bootloader/bl_crypto_client)
```

Comment:
The path is bootloader/image/bl_crypto_client, or did you move the files?

Reply:
No I didn't but I guess I should. Strange, I tried compiling with this option set, and it worked.

Reply:
I say we keep it where it was, to show that it is not used by the botolader, but the images after the bootloader.

Reply:
+1 for moving

Reply:
Ok, moving then 💃